tasks.rb

```diff
@@ -3145,6 +3146,9 @@ def set_hpxml_heating_systems(hpxml_file, hpxml)
     hpxml.heating_systems << hpxml.heating_systems[0].dup
     hpxml.heating_systems[1].id = 'HeatingSystem2'
     hpxml.heating_systems[1].distribution_system_idref = 'HVACDistribution2'
+  elsif ['base-hvac-multiple2.xml'].include? hpxml_file
+    hpxml.heating_systems[1].primary_system = true
+    hpxml.cooling_systems[-1].primary_system = true
```

Comment:
I'd prefer that you just update the original base-hvac-multiple.xml file -- or did you need it to be a different file?

Reply:
There is no real need for `multiple2.xml`.